packages/calcite-components/src/components/stepper/stepper.e2e.ts

```diff
@@ -68,14 +68,29 @@ describe("calcite-stepper", () => {
           <div>Step 4 content</div>
         </calcite-stepper-item>
       </calcite-stepper>`,
-      { display: "grid" }
+      { display: "flex" }
```

Comment:
would this be a breaking change?

Reply:
Visually, it shouldn't but we should at least make a note of this change incase anyone was relying on this element being a grid display.